src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -475,7 +477,9 @@ private async ValueTask<ProtocolToken> ReceiveBlobAsync<TIOAdapter>(TIOAdapter a
 #pragma warning disable 0618
                 _lastFrame.Header.Version = SslProtocols.Ssl2;
 #pragma warning restore 0618
+                _lastFrame.Header.Type = TlsContentType.Handshake;  // Assume. Needed for check bellow.
```

Comment:
What does this comment mean?  Just that we're starting out by assuming it's a handshake but may update it to something else later?
```suggestion
                _lastFrame.Header.Type = TlsContentType.Handshake;  // Assume. Needed for check below.
```

Reply:
We don't have full parsing for SSLv2 format. In some generic case there could be different frame types. But since we use it only during Handshake then the `TlsContentType.Handshake` is assumed. Does it make sense?

Reply:
I updated the comment. Hopefully making it more clear.